xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/attachmentsinline.vm

```diff
@@ -45,7 +45,13 @@ $xwiki.ssfx.use('js/xwiki/viewers/attachments.css', true)
     ## Temporarily disabled, until we fix attachment name handling
     ## <div><label id="xwikiuploadnamelabel" for="xwikiuploadname">$services.localization.render('core.viewers.attachments.upload.filename')</label></div>
     ## <div><input id="xwikiuploadname" type="hidden" name="filename" value="" size="40"/></div>
-    <div class="fileupload-field"><label class="sr-only" for="xwikiuploadfile">$services.localization.render('core.viewers.attachments.upload.file')</label><input id="xwikiuploadfile" type="file" name="filepath" size="40" class="uploadFileInput noitems" data-max-file-size="$!escapetool.xml($xwiki.getSpacePreference('upload_maxsize'))" /></div>
+    <div class="fileupload-field">
+      #if ($services.attachment.configuration.areSummariesEnabled())
+        <label for="xwikiuploadcomment" class="sr-only">$services.localization.render('core.viewers.attachments.upload.comment')</label>
```

Comment:
Why is this label hidden?

Wouldn't it make sense to have a summary per uploaded file instead of one for all files?

Reply:
I only added this label for accessibility purposes, the input itself should hopefully be self-explanatory through the placeholder and the tooltip (granted, it was not in the previous version).

Reply:
As I wrote, placeholders and titles are no replacement for a visible label unless the label is obvious (like a file picker with an upload button). Further, we have standards for forms in XWiki that clearly include a label.